ThunkLibs/libvulkan_device/Guest.cpp

```diff
@@ -1,56 +1,49 @@
-/*
-$info$
-tags: thunklibs|vulkan
-$end_info$
-*/
+#define VK_USE_PLATFORM_XLIB_XRANDR_EXT
+#define VK_USE_PLATFORM_XLIB_KHR
+#define VK_USE_PLATFORM_XCB_KHR
+#define VK_USE_PLATFORM_WAYLAND_KHR
+#include <vulkan/vulkan.h>
 
-#include "Header.inl"
+#include "common/Guest.h"
 
-#include <stdio.h>
-#include <cstring>
-#include <map>
-#include <string>
+#include <cstdio>
 #include <dlfcn.h>
+#include <string_view>
 #include <unordered_map>
-#include <vector>
-
-#include "common/Guest.h"
-#include <stdarg.h>
 
 #include "thunks.inl"
 #include "function_packs.inl"
 #include "function_packs_public.inl"
+#include "symbol_list.inl"
 
 extern "C" {
 static bool Setup{};
 static std::unordered_map<std::string_view,PFN_vkVoidFunction*> PtrsToLookUp{};
 
-static PFN_vkVoidFunction fexfn_pack_vkGetDeviceProcAddr(VkDevice a_0,const char* a_1);
-static PFN_vkVoidFunction fexfn_pack_vkGetInstanceProcAddr(VkInstance a_0,const char* a_1);
-static void fexfn_pack_vkCmdSetBlendConstants(VkCommandBuffer a_0,const float a_1[4]);
-
 // Setup can't be done on shared library constructor
 // Needs to be deferred until post-constructor phase to remove the chance of crashing
 static void DoSetup() {
     // Initialize unordered_map from generated initializer-list
     PtrsToLookUp = {
-#define PAIR(name, ptr) { #name, (PFN_vkVoidFunction*)ptr }
-#include "function_pack_pair.inl"
+#define PAIR(name, unused) { #name, (PFN_vkVoidFunction*)name },
+FOREACH_SYMBOL(PAIR)
 #undef PAIR
     };
 
+#define PAIR(name, unused) printf("SETUP: %s\n", #name);
```

Comment:
Debug message.

Reply:
Fixed, and made another pass over the changes to make sure there's no other random printfs left.